torch/_torch_docs.py

```diff
@@ -2751,10 +2751,9 @@ def merge_dicts(*dicts):
             [-0.0889,  0.2122,  0.1412]])
     >>> a = torch.randn(3, 2, 2) # Example for batched input
     >>> a = a @ a.mT + 1e-03 # make symmetric positive-definite
-    >>> l = torch.linalg.cholesky(a)
-    >>> z = l @ l.mT
-    >>> torch.dist(z, a)
-    tensor(3.5894e-07)
+    >>> u = torch.linalg.cholesky(a)
+    >>> torch.dist(torch.inverse(a), torch.cholesky_inverse(u))
+    tensor(1.9769e-06)
```

Comment:
The previous example was correct, why changing it?

Reply:
The previous example was only a test for `cholesky` function not `cholesky_inverse`. This is an example of batch `cholesky_inverse`  and it tests that results is the same as `torch.inverse`

Reply:
ops, right!